<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizGeneratorGUI</class>
 <widget class="QMainWindow" name="QuizGeneratorGUI">
  <property name="windowTitle">
   <string>Quiz Generator</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>1200</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>9</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="gradeLabel">
            <property name="text">
             <string>Grade:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="gradeCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="topicTree">
          <property name="minimumHeight">
           <number>500</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="headerHidden">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Topics</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="optionsLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="difficultyLabel">
            <property name="text">
             <string>Difficulty:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="difficultyCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Easy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Medium</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hard</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="activityLabel">
            <property name="text">
             <string>Activity Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="activityCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Class Activity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Homework</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Test</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="exportLabel">
            <property name="text">
             <string>Export Format:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="exportFormatCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Google Forms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kahoot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PDF</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>JSON</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="questionTypeLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="qtypeMCQ">
            <property name="text">
             <string>Multiple Choice</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="qtypeMCQSpin"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="qtypeShort">
            <property name="text">
             <string>Short Answer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="qtypeShortSpin"/>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="qtypeTF">
            <property name="text">
             <string>True/False</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="qtypeTFSpin"/>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="qtypeFill">
            <property name="text">
             <string>Fill-in-the-Blank</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="qtypeFillSpin"/>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="qtypeNum">
            <property name="text">
             <string>Numerical</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="qtypeNumSpin"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="addToQueueBtn">
            <property name="text">
             <string>Add to Queue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startGenerationBtn">
            <property name="text">
             <string>Start Generation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveSettingsBtn">
            <property name="text">
             <string>Save Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadSettingsBtn">
            <property name="text">
             <string>Load Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importQuestionsBtn">
            <property name="text">
             <string>Import Questions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportActivitiesBtn">
            <property name="text">
             <string>Export Activities</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="queueStatusLabel">
          <property name="text">
           <string>Status: Idle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="queueTable">
          <property name="maximumHeight">
           <number>150</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <property name="horizontalHeaderLabels">
           <stringlist>
            <string>Title</string>
            <string>Status</string>
            <string>Details</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="questionTable">
          <property name="maximumHeight">
           <number>200</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <property name="horizontalHeaderLabels">
           <stringlist>
            <string>Topic</string>
            <string>Subtopic</string>
            <string>Question</string>
            <string>Type</string>
            <string>Answer</string>
            <string>Actions</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>